modules.go

```diff
@@ -7,5 +7,5 @@ import (
 	_ "github.com/gliderlabs/logspout/routesapi"
 	_ "github.com/gliderlabs/logspout/transports/tcp"
 	_ "github.com/gliderlabs/logspout/transports/udp"
-	_ "github.com/iamatypeofwalrus/logspout-loggly/loggly"
+	_ "github.com/uaudio/logspout-loggly/loggly"
```

Comment:
Thanks for moving so fast on this. Any idea when you think it might merge and pushed to Docker Hub?